server/helpers/swagger.yaml

```diff
@@ -359,6 +359,16 @@ paths:
             summary: Handle GET request for events.
             tags:
                 - EventsAPI
+        post:
```

Comment:
Do we have example requests and responses?

Reply:
I tried generating a few pairs but I kept getting a 405 on making the POST request with thunderclient. 

I tried re-verifying the routes were properly set, but I can't seem to pinpoint where the issue is. Is this normal behavior since the code hasn't been merged yet ? (with the server requiring a cloud provider to run and all)